.scripts/regression-compare.yaml

```diff
@@ -62,12 +62,10 @@ languages:
   - language: python
     outputPath: ../core/test/regression/python
     oldArgs:
-      - --v3
-      - --use:@autorest/python@5.1.0-preview.3
+      - --use:@autorest/python@5.3.2
     newArgs:
       - --version:../core
-      - --v3
-      - --use:@autorest/python@5.1.0-preview.3
+      - --use:@autorest/python@5.3.2
```

Comment:
is it possible to not tie this to the specific version 5.3.2, but just get the latest autorest python?

Reply:
Good catch!  Yep, I'll remove the version.